### ISSUE TITLE
Duplicate interface definition for class 'FlipperClient'

### ISSUE THREAD

**Issue by lijy91** [facebook/flipper#376]
When I try to write a Flipper SDK for Flutter , this error occurred.

![image](https://user-images.githubusercontent.com/3889523/53218284-7eb07a80-3696-11e9-9328-fd0ccf96be38.png)

It uses the pod to manage dependence, this is my pod spec file.

```ruby
flipperkit_version = '0.16.2'

#
# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
#
Pod::Spec.new do |s|
  s.name             = 'flutter_flipperkit'
  s.version          = '0.0.1'
  s.summary          = 'Flipper SDK for Flutter.'
  s.description      = <<-DESC
  Flipper SDK for Flutter.
                       DESC
  s.homepage         = 'https://github.com/lijy91/flutter_flipperkit'
  s.license          = { :file => '../LICENSE' }
  s.author           = { 'JianyingLi' => 'lijy91@foxmail.com' }
  s.source           = { :path => '.' }
  s.source_files = 'Classes/**/*'
  s.public_header_files = 'Classes/**/*.h'
  
  s.dependency 'Flutter'
  s.dependency 'FlipperKit', '~>' + flipperkit_version
  s.dependency 'FlipperKit/FlipperKitLayoutComponentKitSupport', '~>' + flipperkit_version
  s.dependency 'FlipperKit/SKIOSNetworkPlugin', '~>' + flipperkit_version
  s.dependency 'FlipperKit/FlipperKitUserDefaultsPlugin', '~>' + flipperkit_version

  s.xcconfig = {'OTHER_CFLAGS' => '-DFB_SONARKIT_ENABLED=1'}
  s.ios.deployment_target = '9.0'
end
```

My code:
https://github.com/blankapp/flutter_flipperkit/blob/master/ios/flutter_flipperkit.podspec
https://github.com/blankapp/flutter_flipperkit/blob/master/ios/Classes/FlutterFlipperkitPlugin.m

Repo: https://github.com/blankapp/flutter_flipperkit

PS. I have integrated the Android version

**Comment by passy** [comment]
@priteshrnandgaonkar Can you take a look?

**Comment by priteshrnandgaonkar** [comment]
@lijy91, I renamed `FlutterFlipperkitPlugin.m` to `FlutterFlipperkitPlugin.mm`. With this change the above errors went away. But I got `/bin/sh: /packages/flutter_tools/bin/xcode_backend.sh: No such file or directory`, which you will have to look, as it is Flutter related.

I will look into it as to why the extension of the plugin file has to be Objective-C++, as we don't have any c++ in the imported headers. But this fix will unblock you and you can work on your plugin.

Let me know, if you have any flipper related errors and queries.

**Comment by priteshrnandgaonkar** [comment]
Created an another issue for more investigation #378. Closing this issue, as the errors are gone by the above mentioned fix. Feel free to reopen the issue if the same errors reappear. For other issues, create a new one.

**Comment by lijy91** [comment]
thank you for your help.

I renamed `FlutterFlipperkitPlugin.m` to `FlutterFlipperkitPlugin.mm`, which solved my problem. Now my flutter sdk `NetworkPlugin` is already working on android and iOS.
